docker-compose.yml

```diff
@@ -11,6 +11,17 @@ services:
         ports:
             - "5430:5432"
 
+    rserver:
+        build:
+            context: ./rserver
+        environment:
+            - PASSWORD=rstudio_password
+            - POSTGRES_DATALAKE=postgres://postgres:airflow@airflow:5432
```

Comment:
The address provided here is pointing to the airflow metadata database,  the datalake database would look something like `postgres://postgres:airflow@airflow:5432` (or 5430, I am unsure, as Chow is using an alternative port that I am not familiar enough with docker to explain)